thesis-de/content-de.tex

```diff
@@ -1888,16 +1888,104 @@ \section{Userstories}
 Tickets arbeiten können.
 \tm%
 
-\subsection{Userstory 1}
+\subsection{SSD Userstory}
+\\textbf{Name der Story:}\\ Anzeige der SSD's, welche einen Hardwaredeffekt haben könnten.
+\\textbf{Beschreibung:}\\ In der heutigen Zeit, werden vermehrt langsame Hard Disk Drive's
+durch schnellere Solid State Disk's ausgetauscht. Gerade im Serverbetrieb, sorgt
+dies für eine schnellere Absicherung der Daten. SSDs haben jedoch, wie auch
+HDDs einen Lebenszyklus, der bei SSD schneller erreicht werden kann. Der
+Administrator soll hier darüber Benachrichtigt werden, sobald von einer SSD
+ein kritischer Schreib- oder Lesezyklus erreicht worden ist, um die SSD, bereits
+vor dem Ausfall austauschen zu können. Ebenfalls soll hier auch eine Statistik
+ausgegeben werden, welche SSD von welchem Hersteller in der Wahrscheinlichkeit
```

Comment:
Die Statistik können wir liefern? Besitzen wir im Projekt eine Datenbank mit allen Metadaten der physischen Kisten? Also Server200112 in Rack212 besitzt CPU von ... RAM von ... SSD von ... ? Ich glaube das haben wir nicht, bzw haben wir noch nie irgendwo definiert,